awscli/data/codebuild/2016-10-06/completions-1.sdk-extras.json

```diff
@@ -0,0 +1,102 @@
+{
+  "merge": {
+    "resources": {
+      "Projects": {
+        "operation": "ListProjects",
+        "resourceIdentifier": {
+          "project": "projects[]"
```

Comment:
Let's switch this from `project` to `name` as the values returned from this response are project names.